pyproject.toml

```diff
@@ -33,7 +33,7 @@ dependencies = [
 
 [project.optional-dependencies]
 gunicorn = ["gunicorn>=20.1"]  # Lowest which supports wsgi_app in config.
-sentry = ["sentry-sdk[falcon]>=1.5.4"]
+sentry = ["sentry-sdk[falcon]>=1.16.0"]
```

Comment:
Mind adding a comment explaining that we want this for Falcon 3.0 support and linking to https://github.com/getsentry/sentry-python/pull/1733 ?

Reply:
a7a5ff8f7fc919727acffe084622f98036c2abf6